subprojects/packagefiles/imgui/meson.build

```diff
@@ -30,21 +30,30 @@ if dx12_dep.found()
     sources += 'backends/imgui_impl_dx12.cpp'
     dependencies += dx12_dep
 endif
-metal_dep = dependency('metal', required: get_option('metal'))
-if get_option('metal').enabled()
+metal_dep = cpp.find_library('metal', required: get_option('metal'))
```

Comment:
IIRC metal is an apple frameworks library, so if it is added as https://mesonbuild.com/Dependencies.html#appleframeworks it will only ever be found on macOS. Adding it as an ordinary library is probably not the same thing.

Reply:
Done